src/javascript/binary/websocket_pages/user/ico_subscribe.js

```diff
@@ -109,6 +110,31 @@ const ICOSubscribe = (() => {
         $total.html(formatMoney(currency, total));
     };
 
+    const ICOcountDown = () => {
+        const timer = window.setInterval(() => {
+            const start_time = 1510704000; 
+            const current_time = Math.round((new Date()).getTime()/1000); 
+            const time_left = start_time - current_time; 
+            if(time_left >= 0) {
+                $('.timer').removeClass('invisible'); // Make the timer visible.
```

Comment:
Could you get all elements one time only (outside the interval function), so every second it doesn't need to go through DOM again.